client/me/purchases/cancel-purchase/support-box.jsx

```diff
@@ -3,7 +3,27 @@
  */
 import React from 'react';
 
+/**
+ * Internal Dependencies
+ */
+import analytics from 'analytics';
+
 const CancelPurchaseSupportBox = React.createClass( {
+	propTypes: {
+		purchase: React.PropTypes.object.isRequired,
+	},
+
+	trackClickContactSupport() {
+		if ( ! this.props.purchase ) {
```

Comment:
Do we need this line? `purchase` is a required prop, and this is only mounted when the purchase is loaded in `CancelPurchase`.